### ISSUE TITLE
Unsure of what Optional methods are restricted

### ISSUE THREAD

**Issue by Justinn-Paul** [nus-cs2030/2223-s1#169]
Use of restricted Optional methods
Grading terminated.

I got this error, are we not allowed to use Optional?

**Comment by wilsonwid** [comment]
I don't think we are allowed to use `Optional`s in the project since we only started learning about it in Week 7.

**Comment by Wenbo-J** [comment]
I heard from my TA that the banned Optional methods are: isEmpty() isPresent() and get()
clients of Optional cannot invoke these methods

**Comment by nixonwidjaja** [comment]
Yes those 3 are not allowed
